sections/appendix/models.tex

```diff
@@ -0,0 +1,56 @@
+\subsection{Extenstions to our models}\label{subsec:app_exten_models}
+Looking at the results, in \autoref{tab:metric_results}, the Author-Topic and Category-Topic model does not get very high score in Topic Coherence.
+Trying to improve these results, we have run the combination model, using the same setup as the Author-Category model which is explained in \autoref{subsec:combinations}, just exchanging one feature with the document feature.
+Therefore, we create two extra model, Author-Doc and Category-Doc.
+We run them using the same hyperparameters as all the other models and they get similar results to the standard \gls{lda} model.
+
+\begin{table}[h]
+	\centering
+	\caption{Results.}
+	\begin{tabular}{l|c|c|c}
+		Topic Model & Perplexity & \makecell{Topic \\ Coherence} & \makecell{Topic \\ Difference} \\
+		\midrule
+		\Acrlong{lda} & 6761.6 & 0.520 & 0.575 \\
+		Author-Topic Model & 10991.5 & 0.335 & 0.615 \\
+		Category-Topic Model & 11126.0 & 0.370 & 0.560 \\
+		\textbf{Author-Doc Model} & \textbf{888036.4} & \textbf{0.543} & \textbf{0.574} \\
+		\textbf{Category-Doc Model} & \textbf{896045.7} &\textbf{ 0.530} & \textbf{0.575} \\
+	\end{tabular}
+	\label{tab:_extra_metric_results}
+\end{table}
+
+As \autoref{tab:_extra_metric_results} shows, the perplexity score for the two models, are extremely high, since we are multiplying three vectors instead of two.
+When looking at the topics, that the two models produce and comparing them, we can see some suttle difference which might indicate the influence of the metadata.
+\begin{table}
+		\centering
+	\caption{Top 10 words for two topics within the extension models}
+	\begin{tabular}{l|c|c|c|c|c|c|c|c|c|c}
+		Model & 1 & 2 & 3 & 4 & 5 & 6 & 7 & 8 & 9 & 10 \\
+		\midrule
+		Author-Doc & wozniacki & vandt & open & sæt & turneringen & caroline & hobro & runde & nummer & arige \\
+		Category-Doc & vandt & wozniacki & nummer & runde & open & sæt & turneringen & par & slag & dansk \\
+		\midrule
+		Author-Doc & eu & brexit & britiske & storbritannien & aftale & may & parlamentet & sagde & london & premierminister \\
+		Category-Doc & eu & brexit & britiske & storbritannien & parlamentet & may & aftale & europa & london & johnson \\
+		\midrule
+		Author-Doc & natur & dyr & landbrug & naturen & landmænd & skov & hektar & vand & lille & danmarks \\
+		Category-Doc & naturen & natur & du & dansk & hvordan & maske & omradet & landbrug & penge & kystsikring \\
+	\end{tabular}
+	\label{tab:_extra_metric_results}
+\end{table}
+We have taken 3 topic pairs, which seemed to be about the same topics and compared them between the models.
+The first topic pair in \autoref{tab:_extra_metric_results}, is about tennis, since some of the words are talking about Caroline Wozniacki, who is a professional danish tennis player and "turneringen" the tournament.

```

Comment:
```suggestion
The first topic pair in \autoref{tab:top_word_comparison} is about tennis since some of the words are talking about Caroline Wozniacki, who is a professional Danish tennis player, and "turneringen" (the tournament).
```